ecl/eclcc/eclcc.cpp

```diff
@@ -2168,6 +2168,9 @@ void EclCC::processReference(EclCompileInstance & instance, const char * queryAt
         }
     }
 
+    if (isEmptyString(queryAttributePath))
```

Comment:
Would it be clearer if this check/throw be the first statement within EclCC::processReference()? The result is same either way, it just seems like a rejection of a bad argument should happen earlier.